light-curve/README.md

```diff
@@ -355,11 +355,11 @@ where $f(t) -$ flux observation, $A, \gamma , \tau _{rise} , \tau _{fall}  >0$,
 </table>
 
 ### Meta-features
-Meta-features can accept another feature extractors and apply them to pre-processed data.
+Meta-features can accept other feature extractors and apply them to pre-processed data.
 
 #### Periodogram
 
-A number of features based on Lomb–Scargle periodogram as it was an evenly separeated uncertancy-less lime series.
+A number of features are based on Lomb–Scargle periodogram as it was an evenly separated uncertainty-less time series.
```

Comment:
I don't quite understand this sentence, even after adding some words. I thought Lomb-Scargle was for unevenly separated time series?

Reply:
Thanks, I'll change it